packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -97,20 +100,24 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
         queryRunner.manager.getRepository<FieldMetadataEntity>(
           FieldMetadataEntity,
         );
-      const objectMetadata =
-        await this.objectMetadataService.findOneWithinWorkspace(
-          fieldMetadataInput.workspaceId,
-          {
-            where: {
-              id: fieldMetadataInput.objectMetadataId,
-            },
-          },
-        );
+
+      console.time('createOne query');
```

Comment:
It should not, in fact the input is prefilled earlier in the before hook using the context, not the user input @charlesBochet